examples/manager.py

```diff
@@ -0,0 +1,24 @@
+'''
+Waits for a keyboard interupt, and sends a kill signal to the other nodes
+'''
+
+import time
+import zmq
+
+
+def main():
+    context = zmq.Context()
+    shutdown = context.socket(zmq.PUB)
+    shutdown.bind("tcp://*:9000")
+
+    shutdown.send(b'START')
+    try:
+        while True:
+            time.sleep(1)
+    except KeyboardInterrupt:
+        print("Sending kill signal")
+        shutdown.send(b"EXIT")
```

Comment:
I am also unsure on if this should be the mechanism for shutdown, but it may work. It may make more sense to have a heartbeat with `monitor` and have it send a `quit` on that socket.